projects/storefrontapp-e2e-cypress/cypress/integration/regression/my-account/update-password.e2e-spec.ts

```diff
@@ -11,34 +11,21 @@ describe('My Account - Update Password', () => {
     })
   );
 
-  beforeEach(() => {
-    cy.restoreLocalStorage();
-  });
-
-  afterEach(() => {
-    cy.saveLocalStorage();
-  });
-
   verifyAsAnonymous();
-  updatePasswordTest();
-});
 
-describe(`${formats.mobile.width +
-  1}p resolution - Update Password page`, () => {
-  before(() => {
-    cy.window().then(win => win.sessionStorage.clear());
-    cy.viewport(formats.mobile.width, formats.mobile.height);
-  });
+  describe('update profile test for logged in user', () => {
+    beforeEach(() => {
+      cy.restoreLocalStorage();
+    });
 
-  beforeEach(() => {
-    cy.restoreLocalStorage();
-    cy.viewport(formats.mobile.width, formats.mobile.height);
-  });
+    updatePasswordTest();
 
-  afterEach(() => {
-    cy.saveLocalStorage();
-  });
+    afterEach(() => {
+      cy.saveLocalStorage();
+    });
 
-  verifyAsAnonymous();
-  updatePasswordTest();
+    after(() => {
+      login.signOutUser();
```

Comment:
we should add this to the other e2e desktop test 
the after clause with logging them out to make sure

Reply:
the only tests that don't have this is when there's no logged useer:
- close account because the account is de-activated
- update email because last step is verifying that the old email is invalid, so no logged user.